website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -743,45 +767,49 @@ You can specify the following string values for the `MutualTLSMode` field:
 
 List of extensions to modify Envoy proxy configuration. Refer to [Envoy Extensions](/consul/docs/connect/proxies/envoy-extensions) for additional information.
 
-You can configure the following parameters in the `EnvoyExtensions` block:
+The following table describes how to configure values in the `EnvoyExtensions` map:
 
 | Parameter | Description | Data type | Default |
-| ---   | ---         | ---       | ---     |
-| `Name` | Name of the extension. | string | `""` |
-| `Required` | When Required is true and the extension does not update any Envoy resources, an error is returned. Use this parameter to ensure that extensions required for secure communication are not unintentionally bypassed. | string | `""` |
-| `Arguments` | Arguments to pass to the extension executable. | map | `nil` |
+| ---       | ---         | ---       | ---     |
+| `Name` | Specifies the name of the extension. | String | None |
+| `Required` | Specify `true` to require Consul to implement the extension. <p>Use this parameter to ensure that extensions required for secure communication are not unintentionally bypassed.</p><p>Consul returns an error when `Required` is set to `true` and the extension does not update Envoy resources. </p> | String | None |
```

Comment:
Consul doesn't so much _implement_ an extension as apply code that's already there and check whether application is successful; tweaked language to match. Also clarified the effect of errors at runtime.

```suggestion
| `Required` | Specify `true` to require the extension to apply successfully. <p>Use this parameter to ensure that extensions required for secure communication are not unintentionally bypassed.</p><p>Consul logs an error and does not update xDS resources (skipping _all_ extensions) when an extension with `Required` set to `true` fails to apply. </p> | String | None |
```